package.json

```diff
@@ -51,8 +51,9 @@
     "@typescript-eslint/parser": "^6.4.1",
     "browser-sync": "^2.29.3",
     "concurrently": "^8.2.0",
-    "eslint": "^8.47.0",
+    "eslint": "^8.48.0",
```

Comment:
Since ESLint was a bit outdated I figured I might as well bump it to the latest version.